playwright/e2e/sliding-sync/sliding-sync.spec.ts

```diff
@@ -361,52 +305,4 @@ test.describe("Sliding Sync", () => {
         // ensure the reply-to does not disappear
         await expect(page.locator(".mx_ReplyPreview")).toBeVisible();
     });
-
-    test("should send unsubscribe_rooms for every room switch", async ({ page, app }) => {
```

Comment:
This seems still desirable. MSC4186 Simplified Sliding Sync still uses `room_subscriptions` which this was also asserting